utils/choices.py

```diff
@@ -1,9 +1,70 @@
-from django.db.models import IntegerChoices
+from django.db.models import TextChoices, IntegerChoices
 
 class ExampleChoices(IntegerChoices):
     '''
     예시 코드입니다.
     '''
     ONE = 1, '일'
     TWO = 2, '이'
-    THREE = 3, '삼'
\ No newline at end of file
+    THREE = 3, '삼'
+
+class FounderTargetChoices(TextChoices):
+    LOCAL = "local", "동네주민"
+    OUTSIDER = "outsider", "외부인"
+
+class FundingStatusChoices(TextChoices):
+    REVIEW = "REVIEW", "심사중"
+    APPROVED = "APPROVED", "승인됨"
+    REJECTED = "REJECTED", "반려됨"
+    ACTIVE = "ACTIVE", "진행중"
+    CLOSED = "CLOSED", "종료"
+
+
+class RewardCategoryChoices(TextChoices):
+    COUPON = "coupon", "펀딩 할인쿠폰"
+    GIFT = "gift", "펀딩 선물증정"
+    LEVEL = "level", "레벨"
+
+class RewardAmountChoices(IntegerChoices):
+    W5K = 5_000, "5천원"
+    W10K = 10_000, "1만원"
+    W30K = 30_000, "3만원"
+    W50K = 50_000, "5만원"
+
+class PaymentTypeChoices(TextChoices):
+    NORMAL = "NORMAL", "일반 결제"
+    BILLING = "BILLING", "자동결제"
+    BRANDPAY = "BRANDPAY", "브랜드페이"
+
+class PaymentMethodChoices(TextChoices):
+    CARD = "CARD", "카드"
+    VIRTUAL_ACCOUNT = "VIRTUAL_ACCOUNT", "가상계좌"
+    EASY_PAY = "EASY_PAY", "간편결제"
+    MOBILE_PHONE = "MOBILE_PHONE", "휴대폰"
+    TRANSFER = "TRANSFER", "계좌이체"
+    GIFT_CERTIFICATE = "GIFT_CERTIFICATE", "문화상품권"
+    CULTURE_GIFT_CERT = "CULTURE_GIFT_CERT", "도서문화상품권"
+    GAME_GIFT_CERT = "GAME_GIFT_CERT", "게임문화상품권"
```

Comment:
❗ [토스페이먼츠 개발자센터 코어 API 문서](https://docs.tosspayments.com/reference#결제-승인)를 보니, 결제 승인 API가 `"method": "카드"`를 응답한다고 설명하고 있어요. 이런 경우에는 우리 DB에도 토스에서 보내준 응답값을 그대로 저장해야 한다고 생각해요.
```suggestion
class PaymentMethodChoices(TextChoices):
    CARD = "카드"
    VIRTUAL_ACCOUNT = "가상계좌"
    EASY_PAY = "간편결제"
    MOBILE_PHONE = "휴대폰"
    TRANSFER = "계좌이체"
    GIFT_CERTIFICATE = "문화상품권"
    CULTURE_GIFT_CERT = "도서문화상품권"
    GAME_GIFT_CERT = "게임문화상품권"
```